editor/utils/terms.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Can we get some unit tests for this?

Reply:
Added :) they were missing at the time because  I was not certain if a utils function was right for this case.